_episodes/10-data-transformation.md

```diff
@@ -38,9 +38,43 @@ So far we've been looking only at 'String' type data. Much of the time it is pos
 >3. Note how the values are now displayed in green and follow a standard convention for their display format (ISO 8601) - this indicates they are now stored as date data types in OpenRefine. We can now carry out functions that are specific to Dates
 >4. On the Date column dropdown select ```Edit column->Add column based on this column```. Using this function you can create a new column, while preserving the old column
 >5. In the 'New column name' type "Formatted-Date"
->6. In the 'Expression' box type the GREL expression ```value.toString("dd MMMM yyyy")```
+>6. In the 'Expression' box type the GREL expression ```value.toString("dd MMM yyyy")
 {: .checklist}
 
+>## Specifying Date Formatting in GREL Expressions
+>
+>GREL allow us to specify date and time using ```pattern strings```, which are letters that have some specific representation in the function call.
+>
+>Pattern strings are case sensitive, therefore capital and lower case letters have a different meaning and usage.
+{: .callout}
+
+The table below shows letters related to date and time representation.
+
+| Letter| Date or Time Representation|
+| ------------- |:-------------:|
+| `y` | Year|
+| `M` | Month in year|
+| `D` | Day in year|
+| `d` | Day in month|
+| `F` | Day of week in month|
+| `E` | Day name in week|
+| `u` | Day number of week|
+| `a` | AM/PM marker|
+
+The table below presents examples on how to use the patterns as input and the obtained output.
```

Comment:
Do you think it might make sense to use the same date/time input in all the examples so that the outputs are comparable - different formats for the same date/time?

Reply:
Thanks! Yeah I think it is better to have the same date/time. I made the adjustments.